src/main/java/org/ukma/spring/crooodle/dto/BookingDto.java

```diff
@@ -0,0 +1,21 @@
+package org.ukma.spring.crooodle.dto;
+
+import lombok.Getter;
+import org.ukma.spring.crooodle.model.Booking;
+import org.ukma.spring.crooodle.model.User;
+
+import java.time.LocalDate;
+
+@Getter
+public class BookingDto {
+
+    User user;
```

Comment:
DTOs can't use entities or other classes from the business layer directly. If needed - create DTO for them too